src/main/scala/jtag/JtagTap.scala

```diff
@@ -119,8 +114,8 @@ class JtagTapController(irLength: Int, initialInstruction: BigInt)(implicit val
     nextActiveInstruction := irChain.io.update.bits
     updateInstruction := true.B
   } .otherwise {
-    //!!! Needed when using chisel3._ (See #1160)
-    // nextActiveInstruction := DontCare
+    // Needed when using chisel3._ (See #1160)
```

Comment:
is #1160 actually still open?

Reply:
Can we try without this line at all since presumably this should be fixed in FIRRTL now... (@jackkoenig any reason to believe it is still needed)

Reply:
Or, is this "proper" chisel3?

Reply:
Without this line, FIRRTL complains about nextActiveInstruction being not fully initialized -- meaning that in some cases the Wire is never assigned a value.  You need either this or to initialize it with WireInit, for example, to ensure that the Wire is always assigned to something.